scripts/vivisect-py2-vs-py3.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
im not sure this file belongs in the main repo. im glad we have it on a branch/in the history, but i dont think we'll want to test and maintain it in the future. are we ok removing it at some point in the near future?

Reply:
I am ok with removing it. I though it could be useful as it can be easily modified to test run times and compare different versions.